index.html

```diff
@@ -64,11 +64,22 @@ <h2 class="animated fadeInUp">
           <div class="section-title">
             <h2>{% t sections.download.title %}</h2>
           </div>
-          <div class="downloadables">
-            {% for post in site.posts %}
-              {% for doc in post.docs %}
-                {% include post-download-line.html post=post doc=doc %}
+          {% assign postsByYear =
+            site.posts | group_by_exp:"post", "post.date | date: '%Y'" %}
```

Comment:
group by year
the years will be used as filters in a list
and as css classes for each line to be able to show/hide them with a very simple jquery script